src/components/CodeAssignmentModal/index.jsx

```diff
@@ -297,18 +313,8 @@ export class BaseCodeAssignmentModal extends React.Component {
       options.codes = [code.code];
     }
 
-    let pendingEnterpriseUserData;
-    if (hasTextAreaEmails) {
```

Comment:
note `hasTextAreaEmails` gets changed to `validEmails` when i pulled out the logic. that was the bug